cmd/tools/rpcwrappers/main.go

```diff
@@ -275,14 +289,27 @@ func makeGetMatchingClient(reqType reflect.Type) string {
 		// Always route these requests to the same matching node for all namespaces.
 		tq = fieldWithPath{path: "\"not-applicable\""}
 		tqt = fieldWithPath{path: "enumspb.TASK_QUEUE_TYPE_UNSPECIFIED"}
-		nsID = fieldWithPath{path: `"not-applicable"`}
+		nsID = fieldWithPath{path: "\"not-applicable\""}
 	default:
+		tqp = tryFindOneNestedField(t, "TaskQueuePartition", "request", 1)
```

Comment:
oh right, I said this wrong, `request` is the name of the function arg, right?

Reply:
yea, It made me chuckle when I saw the code it generated at first.